CHANGELOG.md

```diff
@@ -31,6 +31,10 @@ time, before giving the error message of why it failed. It's useful for investig
 - Changed: Preset summaries have been split better into categories.
 - Removed: The "Never" option for dangerous actions has been removed from the UI, as it currently doesn't work.
 
+### Custom Elevator Names (Prime 1 + Echoes)
+- Changed: Reworked how elevator custom names are handled in the UI
+- Added: Support for short names in the tracker
```

Comment:
The Changelog is written from the perspective of an end-user. So pure code changes are ignored.
This change would be something like "Added: Prime 1 Elevators in the tracker now have nicknames" or something similar.

Also, this doesn't need a full section to it, should just show up in the general section of the next `## [Unreleased]` block.